meinberlin/apps/plans/templates/meinberlin_plans/plan_create_dashboard.html

```diff
@@ -30,9 +30,18 @@ <h1 class="u-first-heading">{% trans 'Add a new plan' %}</h1>
 
     <div class="l-menu__aside">
         <ul class="progress-line u-spacer-bottom">
-            <li class="progress-line__entry--published">
-                <i class="fas fa-circle progress-line__icon" title="{% trans 'The preview is activated after the first save.' %}" aria-label="{% trans 'The preview is activated after the first save.' %}"></i>
-                <span>{% trans 'The preview is activated after the first save.' %}</span>
+            <li class="progress-line__entry">
+                <span class="progress-line__line--bottom"></span>
+                <i class="fas fa-circle progress-line__icon--active" title="{% trans 'Create plan' %}" aria-label="{% trans 'Create plan' %}"></i>
```

Comment:
Just from a UX point of view I would maybe use `fa-check-circle u-success` here and have `progress-line__icon--active` on the 'Fill in plan details' step as the plan has already been created but I think we merge and check with @mcastro-lqd

Reply:
Yeah, the problem here is that it isn't created. If you leave the page without saving the plan is gone.

Reply:
@phillimorland That answer :point_up:  was for you, not sure you got it without me tagging you?